components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/util/SQLConstants.java

```diff
@@ -57,13 +57,18 @@ public class SQLConstants {
     public static final String REQUEST_ID_COLUMN = "REQUEST_ID";
     public static final String CREATED_BY_COLUMN = "CREATED_BY";
 
+    public static  final String WORKFLOWID_COLUMN = "PARAM_VALUE";
```

Comment:
No need to define this constant. It i already defined. https://github.com/wso2/carbon-identity-framework/pull/4519/files#diff-6b77d285eb536d0c997683f1ea8dd0482bf4d6afbaeeebbf4ef2cc51f3e5dae3R45
Btw try to keep generic when defining a new constant. The variable should have been `PARAM_VALUE_COLUMN` instead of `WORKFLOWID_COLUMN`

Reply:
addressed the comment ref:https://github.com/wso2/carbon-identity-framework/pull/4519/commits/ce9bfa08202eb691d2a4f87aebd30fe3445f0fd5

Reply:
addressed the comment ref:https://github.com/wso2/carbon-identity-framework/pull/4519/commits/ce9bfa08202eb691d2a4f87aebd30fe3445f0fd5